ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1868,6 +1868,21 @@ management_api_gnmi:
   octa:
 ```
 
+#### Management API Models
+
+```yaml
+management_api_models:
+  providers:
+    - name: < provider1 (sysdb | smash) >
+      paths:
+      - path: < path1 >
+        enabled: < true | false | default -> true >
```

Comment:
Wouldn't it be good to use the **disabled** instead of **enabled** to match the cli? @ClausHolbechArista @carlbuchmann 

<img width="394" alt="image" src="https://user-images.githubusercontent.com/88734745/167181304-0d548c21-6969-49c6-b503-c28ec9b66f4d.png">

Reply:
On one side `enabled` matches what we have everywhere in AVD, but here I agree that the CLI spells out `disabled` directly, so we should go with that.

Reply:
👍 will make the model consistent with the CLI